ansible_events_ui/api/__init__.py

```diff
@@ -375,6 +375,16 @@ async def read_project(
     return response
 
 
+@router.delete("/api/project/{project_id}")
+async def delete_project(
+    project_id: int, db: AsyncSession = Depends(get_db_session)
+):
+    query = delete(projects).where(projects.c.id == project_id)
+    await db.execute(query)
+    await db.commit()
+    return
```

Comment:
1. An empty `return` statement is redundant here.

2. However you may consider returning a `204 No Content` status for DELETE operations though.
    > If a DELETE method is successfully applied, the origin server SHOULD send
    > * a [202 (Accepted)](https://www.rfc-editor.org/rfc/rfc9110.html#status.202) status code if the action will likely succeed but has not yet been enacted,
    > * a [204 (No Content)](https://www.rfc-editor.org/rfc/rfc9110.html#status.204) status code if the action has been enacted and no further information is to be supplied, or
    > * a [200 (OK)](https://www.rfc-editor.org/rfc/rfc9110.html#status.200) status code if the action has been enacted and the response message includes a representation describing the status.

    https://www.rfc-editor.org/rfc/rfc9110.html#DELETE


3. In addition to that normally a REST API server should return HTTP 404 if the resource doesn't exist.